src/main/java/it/gov/pagopa/payhub/auth/service/user/IamUserInfoDTO2UserInfoMapper.java

```diff
@@ -60,6 +66,19 @@ public static String buildSystemMappedExternalUserId(String organizationIpaCode)
 	private UserInfo userInfoMapper(IamUserInfoDTO iamUserInfoDTO) {
 		User user = usersRepository.findById(iamUserInfoDTO.getInnerUserId()).orElseThrow(() -> new UserNotFoundException("Cannot found user having inner id:" + iamUserInfoDTO.getInnerUserId()));
 		List<Operator> userRoles = operatorsRepository.findAllByUserId(iamUserInfoDTO.getInnerUserId());
+
+		String orgIpaCode = Optional.ofNullable(iamUserInfoDTO.getOrganizationAccess())
+				.map(IamUserOrganizationRolesDTO::getOrganizationIpaCode)
+				.orElseGet(() -> !userRoles.isEmpty() ? userRoles.get(0).getOrganizationIpaCode() : null);
+
+		Broker brokerInfo = null;
+		if (orgIpaCode != null) {
+			Organization organization = organizationSearchClient.getOrganizationByIpaCode(orgIpaCode, String.valueOf(iamUserInfoDTO.getOrganizationAccess()));
```

Comment:
the second operand is the accessToken! you should retrieve it on the Controller using the SecurityUtils.getAccessToken method, and then adding it to each methods in order to let it arrive here